NSS-Day-Cohort-58/rare-rest-quicksilver-dynamos
# prevented deactivation of admin when only one

## Prevent Loss of All Admins
https://github.com/NSS-Day-Cohort-58/rare-rest-quicksilver-dynamos/issues/11

## Description

Updated function in profile_view.py; now has a check if the active admin count is greater than 1; if not then function checks to see if it is attempting to change is_staff or is_active to False; if so, then no updates are made and an error message is given in the Response; if not, then function updates the user.

## Type of change


To test this branch:
  - Git fetch --all
  - Git checkout nm-minimum-admins
  - [ ] Test A - Test in Postman by making a PUT request to http://localhost:8000/profiles/{id} with raw body: {
        "user": {
            "is_staff": false,
            "is_active": true
        }
    }
    Choose {id} of different profiles until you have only on with is_staff set to true; then try the remaining {id} to see the error message.
  - [ ] Test B - Reseed database with ./seed_databse.sh
        Test in Postman by making a PUT request to http://localhost:8000/profiles/{id} with raw body: {
        "user": {
            "is_staff": true,
            "is_active": false
        }
    }
    Choose {id} of different profiles until you have only on with is_active set to true; then try the remaining {id} to see the error message.

## Checklist:
